.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+
+NOTE: IF YOU HAVE A *QUESTION*, PLEASE POST IT ON THE OKTA DEVELOPER FORUM (https://devforum.okta.com) INSTEAD. ISSUES IN THIS REPOSITORY ARE RESERVED FOR BUG REPORTS AND FEATURE REQUESTS.
+-->
+
+## I'm submitting this issue for the package(s):
+- [ ] jwt-verifier
+- [ ] okta-angular
+- [ ] oidc-middleware
+- [ ] okta-react
+- [ ] okta-react-native
+- [ ] okta-react-native
+- [ ] okta-vue
```

Comment:
Instead of listing out the possibilities here, would it make more sense to have a developer specify which lib in the Issue title?

`okta-angular: Doesn't do XYZ`

Reply:
I think it makes less effort for them to just click a checkbox, and then we add the label accordingly, which shows up also in the title but with cool colors :D

Also (even if probably a rare case) allows to select more options.